CRM/ActivityTypeACL/BAO/ACL.php

```diff
@@ -75,7 +75,11 @@ public static function getPermissionedActivities(&$activities = NULL, $action =
         unset($activities[$actTypeId]);
       }
       elseif ($label) {
-        $activities[$actTypeId] = CRM_Core_PseudoConstant::getLabel('CRM_Activity_BAO_Activity', 'activity_type_id', $actTypeId);
+        $activities[$actTypeId] = civicrm_api3('OptionValue', 'getvalue', [
```

Comment:
@Edzelopez is there a reason to use this over the PseudoConstant function call?

Reply:
Yes, the getLabel was conflicting with the ACL code and failing to return a label, with it only returning an activityID in an array.

Reply:
It would very likely be more appropriate for you to fix the bug in the pseudoconstant fn than to avoid using it.